test/python/circuit/test_rust_equivalence.py

```diff
@@ -21,7 +21,7 @@
 from qiskit.circuit import QuantumCircuit
 from qiskit.circuit.library.standard_gates import get_standard_gate_name_mapping
 
-SKIP_LIST = {"rx", "ry", "ecr"}
+SKIP_LIST = {"rx", "ry", "ecr", "r"}
```

Comment:
Why add the `r` gate to the SKIP_LIST? All others haven't been implemented yet, but `r` has an implementation.

Reply:
Good question. I added and removed several times while debugging. I'll recheck.

Reply:
Fixed in [`7eef0f3` (#12662)](https://github.com/Qiskit/qiskit/pull/12662/commits/7eef0f3d8292f12ebbdcb04b1fb3834e5141ea1d). It failed because the decomposition had an error.